AllReadyApp/Web-App/AllReady/Features/Notifications/NotifyVolunteerForUserUnenrolls.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using AllReady.Areas.Admin.Features.Activities;
+using AllReady.Features.Login;
+using AllReady.Models;
+using MediatR;
+using Microsoft.Extensions.OptionsModel;
+
+namespace AllReady.Features.Notifications
+{
+    public class NotifyVolunteerForUserUnenrolls : INotificationHandler<UserUnenrolls>
+    {
+        private readonly IMediator _bus;
+        private readonly IOptions<GeneralSettings> _options;
+
+        public NotifyVolunteerForUserUnenrolls(IMediator bus, IOptions<GeneralSettings> options)
+        {
+            _bus = bus;
+            _options = options;
+        }
+
+        public void Handle(UserUnenrolls notification)
+        {
+            var model = _bus.Send(new ActivityDetailForNotificationQuery {ActivityId = notification.ActivityId});
+
+            var signup = model.UsersSignedUp?.FirstOrDefault(s => s.User.Id == notification.UserId);
+
+            if (string.IsNullOrWhiteSpace(signup?.PreferredEmail)) return;
```

Comment:
Fall back to `User.Email` if no `PreferredEmail` (potentially also require `User.EmailConfirmed` but I don't think that's strictly necessary yet)